wgpu-hal/src/metal/adapter.rs

```diff
@@ -853,17 +853,22 @@ impl super::PrivateCapabilities {
         }
     }
 
-    pub fn features(&self) -> wgt::Features {
+    pub fn features(&self, device: &mtl::Device) -> wgt::Features {
```

Comment:
instead of passing the device here, it should have a field like `supports_depth_clamping`, which is initialized earlier, like every other thing in here

Reply:
Done